### ISSUE TITLE
install?

### ISSUE THREAD

**Issue by adamf663** [Zimbra-Community/reply-by-filter#2]
You have great instructions on how to use the zimlet.

is there any possibility you might document how to install it?

**Comment by ghost** [comment]
Hello,

Can you please help test the release:

      su zimbra
      wget https://github.com/Zimbra-Community/reply-by-filter/releases/download/0.0.1/tk_barrydegraaff_reply_by_filter.zip -O /tmp/tk_barrydegraaff_reply_by_filter.zip
      zmzimletctl deploy /tmp/tk_barrydegraaff_reply_by_filter.zip

If it works, I will add the instruction to the readme.

**Comment by adamf663** [comment]
It installed perfectly.
It doesn't show up on the preferences/zimlets list.   Minor point;  I can figure out the zimbra commands to remove it if ever I want to.
Much worse is that created filters do not show up on the preferences/filters list.  I refreshed the page in severall ways and the new filter never showed up.
Wondering if it simply wasn't created, I created another filter with the same name and got the error message that a filter by that name already existed.

----- Original Message -----
